README.md

```diff
@@ -64,3 +64,16 @@ Press CTRL+C to quit
  * Debugger PIN: 113-666-066
 ```
 Now visit [`http://localhost:5000/`](http://localhost:5000/) in your web browser to view the app.
+
+## Running the test
+Run `poetry add pytest` to install pytest in your project
```

Comment:
Worth noting that this is only required the first time; after that it's recorded in your `pyproject.toml` & `poetry.lock` files so that future developers will pick this up automatically when they run `poetry install`